src/VkCssOptimize.php

```diff
@@ -322,7 +322,7 @@ public static function get_html_start( $is_use_themes ) {
 		// template_redirect が呼ばれる前でのみ実行する .
 		if ( $is_use_themes && did_action( 'template_redirect' ) === 0 ) {
 			// バッファ開始.
-			ob_start( 'self::css_tree_shaking_buffer' );
+			ob_start( array( VkCssOptimize::class, 'css_tree_shaking_buffer' ) );
```

Comment:
```suggestion
			ob_start( array(  __CLASS__, 'css_tree_shaking_buffer' ) );
```
@drill-lancer これでよくね（・ｗ・？

Reply:
@kurudrive 
そうかも知れません。
AI に相談した結果上記の回答が得られたのですがどちらでも良いようです。